contrib/test/integration/system.yml

```diff
@@ -1,21 +1,63 @@
 ---
 
-- name: register a repo for Golang
-  yum_repository:
-    name: 'centos-paas-sig-openshift-origin37-rpms'
-    state: present
-    description: 'CentOS PaaS SIG Origin 3.7 Repository'
-    baseurl: 'https://cbs.centos.org/repos/paas7-openshift-origin37-candidate/x86_64/os/'
-    includepkgs: 'golang'
-    gpgcheck: no
-    sslverify: no
-  when: ansible_distribution in ['RedHat', 'CentOS']
-
 - name: Make sure we have all required packages
   package:
     name: "{{ item }}"
     state: present
-  with_items: "{{ rpm_pkgs }}"
+  with_items:
+    - btrfs-progs-devel
```

Comment:
This package will no longer be in RHEL 7.4.
Why are you reverting this?

Reply:
The commit moves it out of `vars` since there's only one place the list is used and it's hard to see what is happening when you install RPMs here. If you'd rather me keep it in `vars` I can. No strong feelings here.

Re: `btrfs-progs-devel` -- I did not remove or add this, just moved it from one place to the other. I was able to install it with `rhel-server-releases-optional` enabled.I can see from my current testing the following are missing in 7.4:

```
npm
python2-mock
python-click
```

I don't really understand why we have those deps at all.
```

Reply:
@rhatdan The deps are just being moved around here not re-added. 
@stevekuznetsov I think we should be fine without those dependencies unless ansible needs them for some reason.

Reply:
Ok no problem,  @stevekuznetsov, I think @cevich just moved it out, but if it is only used in one place I am fine with moving it.  I have a pull request to remove btrfs from RHEL, but we can wait for that until we have a RHEL7.4 AMI.